trakt_scrobbler/trakt_interface.py

```diff
@@ -40,6 +40,12 @@ def get_trakt_id(title, item_type, year=None):
 
     logger.debug(f'Searching trakt: Title: "{title}"{year and f", Year: {year}" or ""}')
     results = search(title, [required_type], year)
+    if (results == [] and year is not None): # no match, possibly a mismatch in year metadata
+        msg = f'Trakt search yielded no results for the {required_type}, {title}, Year: {year}. Retrying search without filtering by year'
+        logger.warning(msg)
+        notify(msg, category="trakt")
+        results = search(title, [required_type]) # retry without 'year'
```

Comment:
Two spaces before the comment.
```suggestion
        results = search(title, [required_type])  # retry without 'year'
```